CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 * [Feature] Adds Redshift support for `%sqlplot boxplot`
 * [Fix] Fix error when using SQL Server with pyodbc that caused queries to fail due to multiple open result sets
 * [Fix] Improves performance when converting DuckDB results to `pandas.DataFrame`
+* [Fix] Opengraph improvements (#253)
```

Comment:
write a more detailed changelog message, and this should have the `[Doc]` prefix